ci/docker/android-builder/Dockerfile

```diff
@@ -61,6 +45,23 @@ RUN mkdir $HOME/.android && \
     "extras;google;google_play_services" \
     "extras;google;m2repository" && \
   chmod -R o+rwX ${ANDROID_HOME}
+
+# --------------- Gradle Profiler -----------------
+# https://github.com/gradle/gradle-profiler/releases
+ARG PROFILER_VERSION=0.12.0
```

Comment:
https://github.com/gradle/gradle-profiler/releases/tag/v0.12.0

Important changes for us:
- Update the gradle-profiler for Gradle 6.5-rc-1 and the configuration cache
- Allow scenarios to use different commands than build